app/controllers/password_resets_controller.rb

```diff
@@ -1,5 +1,5 @@
 class PasswordResetsController < ApplicationController
-  skip_before_action :authorize
+  skip_before_action :login_authorization
```

Comment:
Is this naming convention current rails standard for `login authorization` (as opposed to `authorize`)?

Reply:
Hmm I was trying to differentiate between login_authorization and admin_authorization, as these provide different levels of restrictions. But I guess since all admin_authorization only occurs in the admin section, this might be redundant. Will revert.